tracer.go

```diff
@@ -151,7 +155,6 @@ func NewTracer(
 			"128 bit trace IDs, consider enabling the \"Gen128Bit\" option")
 	}
 	t.process = Process{
-		UUID:    "PLACEHOLDER", // TODO
```

Comment:
the whole point of this field was to allow all the structs that implement ProcessSetter to use this client UUID. Yes it's unfortunate that we have to add the client uuid to both the tags and this field but I prefer passing the UUID as a first class variable rather than having all the ProcessSetter's iterate over tags to retrieve the client UUID.

Reply:
Ya considered both. Essentially, I want the propagation through tags but the in process representation to be cached in a field. Wasn't sure if a field was preferred over a method. Up to you.

Reply:
let's use the field. Since the ProcessSetter has already been released, removing UUID is a breaking change.